docs/management_api.yml

```diff
@@ -120,22 +154,59 @@ paths:
         500:
           $ref: '#/components/responses/InternalServerError'
 
-  /devices/{id}/check-update:
+  /devices/{id}/download:
     post:
       tags:
         - ManagementAPI
-      operationId: Check Update
-      summary: Trigger check-update for the Mender client running on the device
+      operationId: Download
+      summary: Download a file from the device
       parameters:
         - in: path
           name: id
           required: true
           schema:
             type: string
           description: ID for the target device.
```

Comment:
minor: "target device" somehow I would change to "device" or to "device to get the file from" (feel free to ignore)